### DISCUSSION TITLE
how to ignore .svn  direcotry when searching for file

### DISCUSSION THREAD

**Discussion by resreqgh01** [ibhagwan/fzf-lua#1131] [General]
I am a new user to fzf-lua and try to use lua require('fzf-lua').files to search for files under current directoy, however it is super slow when there is .svn directory. How can I configure to exclude certain directories while searching?

THanks
-James

**Comment by ibhagwan** [comment]
You can change the underlying command to ignore certain globs, if you have `fd` is installed you can modify `files.fd_opts` to ignore the svn folder, for example:
```lua
:lua require'fzf-lua'.files({ cmd = "fd --color=never --type f --hidden --follow --exclude .svn" })
```

**Reply by resreqgh01** [reply]
Hi ibhagawan,

Thanks a lot for the quick response. After adding the configuration, it won't show .svn directory any more .
```
	config = function()
			-- calling `setup` is optional for customization
			require("fzf-lua").setup({
				files = {
					fd_opts=" --color=never --type f --hidden --follow --exclude .svn --exclude .git"
				}

			})
```

Not directly related to this question, it stills takes some time to get the file list in certain directories  with the icon splining, the parent directies(not direct parent) inclue .git directories.   If I move the working directory to my home directory, the list comes up almost immediatly.   Wondering what is going on while the icon spinning...

Thanks
-James

**Reply by ibhagwan** [reply]
> Wondering what is going on while the icon spinning...

That's probably the `git_icons` setting, it conviniently shows git indicators next to the files (i.e. `M` for modified, etc), in large git repositories that `git status` command can taka a while,  set `files.git_icons = false`:

```lua
:lua require'fzf-lua'.files({ git_icons = false })
```
